tests/std/include/test_atomic_wait.hpp

```diff
@@ -43,7 +43,7 @@ void test_atomic_wait_func_impl(UnderlyingType& old_value, const UnderlyingType
         // timing assumption that the main thread evaluates the `wait(old_value)` before this timeout expires
         std::this_thread::sleep_for(waiting_duration);
         add_seq('6');
-#endif // CAN_FAIL_ON_TIMING_ASSUMPTION

```

Comment:
I'm not sure why this is being removed (more than 1 line is guarded) but as there are no nearby preprocessor guards, no change requested.